pom.xml

```diff
@@ -282,7 +274,7 @@
         <version.org.google.guava>30.0-jre</version.org.google.guava>
 
         <!-- >>> Spring integration tests -->
-        <version.org.springframework.boot>2.3.4.RELEASE</version.org.springframework.boot>
+        <version.org.springframework.boot>2.4.0-RC1</version.org.springframework.boot>
```

Comment:
`2.4.0` is out

Reply:
Right, I was looking for 2.4.0.RELEASE and couldn't find it, but it seems they dropped the `.RELEASE` suffix.
I updated the PR, thanks for the heads-up.